.circleci/config.yml

```diff
@@ -41,24 +41,52 @@ jobs:
         - ./chart-operator
         - ./architect
 
-  e2eTestBasicPR:
+  e2eTestCurBasicPR:
```

Comment:
Diff looks a bit strange, but I added the jobs aligned with `aws-operator` here.